lib/mixins/host-attributes-mixin.html

```diff
@@ -0,0 +1,65 @@
+<!--
+@license
+Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<link rel="import" href="property-accessors.html">
+<link rel="import" href="../utils/mixin.html">
+<script>
+  (function() {
+    'use strict';
+
+    /**
+     * Element class mixin to provide default attributes and values, similar to
+     * the "hostAttributes" configuration property used in Polymer 1.x.
+     *
+     * By returning a map from a static class property named "hostAttributes"
+     * , the custom element will ensure that the given attributes are set to given value if
+     * that attribute has not previously been set.
+     *
+     * For example, if an element implements hostAttributes like so:
+     *
+     * ```
+     *   class Foo extends Polymer.HostAccessors(Polymer.Element) {
+     *     static get hostAttributes() { return {'foo': 'bar' }}
+     *   }
+     * ```
+     *
+     * An instance of that element will have an attribute named "foo"
```

Comment:
I forgot to read this sentence because it was too spaced out, and too close to the type (so i assumed it was a compiler thing.

How about "For example, if an element implements `hostAttributes` like below, then an instance of that element will boot up with an attribute named "foo"`